doc/enterprise/stats.md

```diff
@@ -0,0 +1,85 @@
+# Advanced Statistics
+
+To take advantage of the advanced statistics features in Pachyderm Enterprise Edition, you need to:
+
+1. Run your pipelines on a Pachyderm cluster that has activated Enterprise features (see [Deploying Enterprise Edition](deployment.html) for more details).
+2. Enable stats coolection in your pipelines by including `"enable_stats": true` in your [pipeline specifications](http://pachyderm.readthedocs.io/en/latest/reference/pipeline_spec.html#enable-stats-optional).
+
+You will then be able to access the following information for any jobs corresponding to your pipelines:
+
+- The amount of data that was uploaded and downloaded during the job and on a per-datum level.
+- The time spend uploading and downloading data on a per-datum level.
+- The amount of data uploaded and downloaded on a per-datum level.
+- The total time spend processing on a per-datum level.
+- Success/failure information on a per-datum level.
+- The directory structure of input data that was seen by the job.
+
+The primary and recommended way to view this information is via the Pachyderm Enterprise dashboard, which can be deployed as detailed [here](deployment.html#deploying-the-pachyderm-enterprise-edition-dashboard). However, the same information is available through the `inspect-datum` and `list-datum` `pachctl` commands or through their language client equivalents.  
+
+**Note** - We recommend enabling stats for all of your pipeline and only disabling the feature for very stable, long-running pipelines. The persisted stats are persisted minimal in size in comparison to the actual size of your data, and none of your data is duplicated in producing the stats.
```

Comment:
awkward wording "persisted stats are persisted minimal in size in comparison". 

This statement also isn't completely true if you datum size is small. Stats can end up being a meaningful amount of data. The part about pachyderm deduping everything is definitely true and great to keep mentioned here.

@derekchiang In a scenario where a user wants to reclaim space, how easy would it be for them to do delete-commit on the stats branch after successful datums and only keep around failed ones? This is probably something we'll need to implement automatically later if users actually want it.